bazel/linux/bundles.bzl

```diff
@@ -102,6 +107,17 @@ You can set up the bundle like this:
         "run_args": attr.string(
             doc = "Optional parameters to pass to the run_bin. Can use shell expansion.",
         ),
+        "cleanup_cmds": attr.string_list(
+            doc = "Shell commands to run OUTSIDE the VM AFTER the RUN (in reverse order) to clean up the environment (before cleanup_bin - optional)",
```

Comment:
can you expand the docstring here (or a better place? not sure) to explain the difference between check and cleanup?

On the top of my head:
- cleanups are ALWAYS run
- checks are only run if the tests in the VM complete successfully? and in non-interactive mode?

Otherwise, when would one use cleanup vs check?

Reply:
Fixed